integration/src/integration.ts

```diff
@@ -21,10 +21,6 @@ import { terraTests } from "./terra";
 import { kavaTests } from "./kava";
 import { WalletSuite } from "./wallets/suite";
 
-import { ethereum } from "./wallets/mocks/@metamask/detect-provider";
-
-jest.mock("@metamask/detect-provider", () => async () => Promise.resolve(ethereum));
-
```

Comment:
This was mocking `@metamask/detect-provider` during every wallet's integration test, not just MM.